### ISSUE TITLE
Input text, timers.

### ISSUE THREAD

**Issue by rul3zz** [benct/lovelace-multiple-entity-row#65]
Hello and tnx for this amazing integration.
I'm trying to use it to have in a single line the possibility to write a timer (from a input_text) and read the remaining time (from a entity time.* ).  Unfortunately after setting it, there's no place to write the input_text and the remaining timer is not show (just idle/active).
Attached an example
Any suggestion?

![Screenshot_5](https://user-images.githubusercontent.com/17640550/78342447-98123e80-7599-11ea-8a03-01d1579ad5ed.png)

**Comment by cwildfoerster** [comment]
Yes, would be nice to be able to use inputs, like date/time for alarms etc.

**Comment by kazanjig** [comment]
Similarly, it would be nice to be able to use a slider as well

![image](https://user-images.githubusercontent.com/29441832/84393273-646f1880-abc9-11ea-8f3a-1af8e1da906d.png)

**Comment by Tenderlaan54** [comment]
Yes, same problem here. hope to see this

**Comment by Fabenissimo** [comment]
It would be great to be able to see the remaining time of the timer and not the actual state (idel/active/paused/...) (If you add a timer to a stock entites-lovelace card you don't get the actual state but the remaining time of the timer). Currently, there's no workaround to show the remaining time of a timer in your card that I'm aware of. The attribute "remaining" only get's updated once you pause the timer.

**Comment by SAOPP** [comment]
Oh yeah guys, timers It what I need

**Comment by kitkat270** [comment]
Love this, great for filling up wasted space in the UI. But it'd be even better if it could show timer remaining time, not just active/idle.

Thank you :-)

**Comment by navycrow** [comment]
+1

**Comment by maximatchev** [comment]
+1
I need it too

**Comment by beccsjb** [comment]
+1 it would be amazing

**Comment by mgiako** [comment]
+1
I need it too

**Comment by pnsiva** [comment]
+1
I need it too

**Comment by booroondook-eng** [comment]
![2024-04-09 21_55_26-Window](https://github.com/benct/lovelace-multiple-entity-row/assets/8210458/341e30c2-bda0-4511-b701-4ca714cbf17e)
+1

**Comment by mwb9aa** [comment]
+1
I would like this feature as well. I'm content with a workaround, but have no idea how to achieve it.

**Comment by antgustech** [comment]
> ![2024-04-09 21_55_26-Window](https://private-user-images.githubusercontent.com/8210458/320971303-341e30c2-bda0-4511-b701-4ca714cbf17e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1NTE1MDYsIm5iZiI6MTczNTU1MTIwNiwicGF0aCI6Ii84MjEwNDU4LzMyMDk3MTMwMy0zNDFlMzBjMi1iZGEwLTQ1MTEtYjcwMS00Y2E3MTRjYmYxN2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMzBUMDkzMzI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Yjk1NTNiZTQ5NDBiOTljNjY4YTcwNDg2MDI5YmVlMWY4Mjc2NTFmZmQyODM5OTdmMTNmODQ1ODYyYmFjYTU3YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.lmu28V0_1DMTh2QT-IghGymOrIMUkJWCOheZIN8krLg) +1

I have this issue as well!

**Comment by jpettitt** [comment]
The countdown half of this is in [4.10.0-alpha.2](https://github.com/benct/lovelace-multiple-entity-row/releases/tag/v4.10.0-alpha.2) — a `timer` entity now shows its remaining time rather than just active/idle. The other half of your question, editing an `input_text` inline from the row, isn't something the card does; HA's own `input_text` row handles that. In HACS, enable _Show beta versions_ to try it. Closing with the 4.10.0 release.

**Comment by jpettitt** [comment]
Released in [4.10.0](https://github.com/benct/lovelace-multiple-entity-row/releases/tag/v4.10.0) — a `timer` entity now shows its remaining time rather than just active/idle. The other half of your original question, editing an `input_text` inline from the row, remains out of scope; HA's own `input_text` row handles that. Closing this after five years — thanks for the original report.
